pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[tool.mypy]
+check_untyped_defs = "true"
+disallow_any_generics = "true"
+disallow_incomplete_defs = "true"
+disallow_untyped_defs = "true"
+no_implicit_optional = "true"
+
+[tool.poetry]
+name = "qe-metrics"
+version = "1.0.0"
+description = "Report QE metrics"
+authors = ["Caleb Evans <caevans@redhat.com>"]
+readme = "README.md"
+license = "Apche-2.0"
+homepage = "https://github.com/RedHatQE/qe-metrics"
+repository = "https://github.com/RedHatQE/qe-metrics"
+packages = [{include = "qe_metrics"}]
+include = [{path = "manifests/*"}]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+]
+
+[tool.poetry.urls]
+Download = "https://github.com/RedHatQE/qe-metrics"
+"Bug Tracker" = "https://github.com/RedHatQE/qe-metrics/issues"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+click = "^8.1.7"
+jira = "^3.5.2"
+jinja2 = "^3.1.2"
+python-simple-logger = "^1.0.8"
+psycopg2-binary = "^2.9.9"
+pony = "^0.7.17"
+
+[tool.poetry.group.dev.dependencies]
+tox = "^4.11.4"
+pre-commit = "*"
```

Comment:
any reason to not pin `pre-commit` version?
did `poetry add pre-commit` failed?